src/borg/crypto/_crypto_helpers.c

```diff
@@ -1,36 +1,9 @@
-/* some helpers, so our code also works with OpenSSL 1.0.x */
+/* some helpers, so our code also works with LibreSSL */
 
-#include <string.h>
 #include <openssl/opensslv.h>
-#include <openssl/hmac.h>
 
```

Comment:
Needs `#include <openssl/evp.h>` because of `EVP_CIPHER` below.

Reply:
done.